operator/pkg/apis/operator/v1alpha1/type.go

```diff
@@ -66,7 +66,7 @@ type KarmadaSpec struct {
 
 	// FeatureGates enabled by the user.
 	// - Failover: https://karmada.io/docs/userguide/failover/#failover
-	// - GragscefulEviction: https://karmada.io/docs/userguide/failover/#graceful-eviction-feature
+	// - GracefulEviction: https://karmada.io/docs/userguide/failover/#graceful-eviction-feature
```

Comment:
Hi! There is a similar [typo](https://github.com/karmada-io/karmada/blob/master/operator/config/crds/operator.karmada.io_karmadas.yaml#L1460) to fix.

Reply:
That's the auto-generated file that is already included in this PR.